api/streams.go

```diff
@@ -206,15 +205,25 @@ func (c *StreamChannelOf[StreamType, MsgType]) sendLoop(s *StreamsOf[StreamType,
 			return
 		}
 
-		// Some protobuf refletion magic to print more meaningful debug messages. Since this causes some performance
-		// drawbacks, we are only doing this if DEBUG level is enabled.
-		if logrus.IsLevelEnabled(logrus.DebugLevel) {
-			id := extractID(m)
-			if id != "" {
-				s.log.Debugf("%T containing id %s sent to %s (%s)", m, id, c.component, c.target)
-			} else {
-				s.log.Debugf("%T sent to %s (%s)", m, c.component, c.target)
-			}
+		// We want to log some additional information about this stream and its payload
+		preq, ok := any(m).(PayloadRequest)
+		if !ok {
+			return
+		}
+
+		// Retrieve the payload from the request. The request itself is usually
```

Comment:
Lines 214-226 are also done in `LogRequest`. Maybe we can even call LogRequest here?! with the `sent to xxx` part as params? @anatheka

Reply:
I did it as in your suggestion. However, I changed the logging message again from `*assessment.Metric updated with ID 'TransportEncryptionEnabled'.` to `*assessment.Metric with ID 'TransportEncryptionEnabled' updated.` so that it fits. 

Furhermore, I had to move api/request.go to api/request/request.go due to import cycle errors from api/streams.go and internal/logging/logging.go.

@oxisto please check if it fits.

Reply:
I tried to solve the circular dependency in a different way, similar to how protobuf does it. The actual definition of the interface is now moved to an internal package (which gives us more freedom) and the public `api` package just contains a type alias.